tortoise/backends/mysql/client.py

```diff
@@ -76,6 +79,8 @@ def __init__(
         self._connection = None
 
     async def create_connection(self, with_db: bool) -> None:
+        if charset_by_name(self.charset) is None:  # type: ignore
+            raise FieldError(f"Unknown charset {self.charset}")
```

Comment:
Why `FieldError`? Would it not make more sense to raise `DBConnectionError`? As this is a failure when setting up a connection?

Reply:
fixed👍